usr/share/rear/layout/save/default/450_check_bootloader_files.sh

```diff
@@ -27,6 +27,9 @@ case $used_bootloader in
         #  - GRUB2 : SLES >= 12, RHEL >= 7, Ubuntu and other new Linux on POWER distro.
         CHECK_CONFIG_FILES=( ${CHECK_CONFIG_FILES[@]} /etc/lilo.conf /etc/yaboot.conf /etc/grub.cfg /etc/grub2.cfg /boot/grub2/grub2.cfg /boot/grub/grub.cfg)
         ;;
+    (ARM|ARM-ALLWINNER)
```

Comment:
This looks very similar to the code in `455_save_allwinner_bootloader.sh`, why not expand the `CHECK_CONFIG_FILES` variable there as well.

As ARM bootloader can only be used on arm devices I don't see the need to have this in the generic code path (this BTW holds true also for PPC and LILO).

This is a question of taste. One taste could be to easily see what other bootloaders check for and therefore put it here together. Other taste could be to have all plattform-specific stuff in corresponding subdirs like `arm`

Reply:
This is needed here, because otherwise it would Error out.